joomla/joomla-cms
# [4.0] Add a parameter for "back-to-top" button in Cassiopeia

Pull Request for Issue #25239 .

### Summary of Changes
Added a parameter to show / hide the back-to-top button in Cassiopeia

### Testing Instructions

Install PR, run npm ci or compile the scss files from Cassiopeia
Go to template, under "advanced" a new switcher should appear
![Screenshot_2020-08-21 Templates Edit Style - Joomla Cassiopeia - Administration](https://user-images.githubusercontent.com/9153168/90889405-6a972780-e3b8-11ea-8906-962470595333.png)


### Actual result BEFORE applying this Pull Request

Currently the footer appears when a module has been published on position "footer". And the back-to-top only appears if the footer is visible. Since the footer is part of the grid, it is not easy to move the back-to-top outside the grid.


### Expected result AFTER applying this Pull Request

Now it is possible to decide to have a back-to-top or not. If yes, the footer will be rendered showing the button, independently if there is a module in the position footer.
It is also possible to show only modules (switch back-to-top off) or modules and back-to-top (switch on).

Example with module on footer and back-to-top button

![Screenshot_2020-08-21 Home](https://user-images.githubusercontent.com/9153168/90889516-9e724d00-e3b8-11ea-8a70-acd90a968b98.png)


### Documentation Changes Required
